drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_util.c

```diff
@@ -368,6 +368,20 @@ static int nrf_wifi_util_tx_stats(const struct shell *shell,
 	return 0;
 }
 
+static int nrf_wifi_util_show_rpu_ps_state(const struct shell *shell,
+					   size_t argc,
+					   const char *argv[])
+{
+	int rpu_ps_state = -1;
+
+	wifi_nrf_fmac_get_rpu_ps_state(ctx->rpu_ctx, &rpu_ps_state);
+
+	shell_fprintf(shell,
+		      SHELL_INFO,
+		      "rpu_state = %s\n", rpu_ps_state ? "AWAKE" : "SLEEP");
```

Comment:
The print can be more user friendly, as in: `"Sleep state = %s\n", rpu_ps_state ? "AWAKE" : "SLEEP"`